spec/system/admin_customize_emojis_spec.rb

```diff
@@ -0,0 +1,33 @@
+#frozen_string_literal: true
+
+describe "Admin Customize Emoji Page", type: :system do
+  fab!(:current_user) { Fabricate(:admin) }
+
+  let(:emojis_page) { PageObjects::Pages::AdminEmojis.new }
+  let(:dialog) { PageObjects::Components::Dialog.new }
+  let(:settings_page) { PageObjects::Pages::AdminSiteSettings.new }
+
+  before do
+    Fabricate(:custom_emoji)
+
+    sign_in(current_user)
+  end
+
+  it "shows a list of custom emojis" do
+    emojis_page.visit_page
+    expect(emojis_page).to have_emoji_listed("joffrey_facepalm")
+  end
+
+  it "can delete a custom emoji" do
+    emojis_page.visit_page
+    emojis_page.delete_emoji("joffrey_facepalm")
```

Comment:
```suggestion
    emojis_page.visit_page.delete_emoji("joffrey_facepalm")
```

Reply:
Not saying this is technically wrong, but we've never done this in any other system spec from what I can see. IMO having visit as the separate step is totally fine. But, happy to go with whatever Ted decides here, it's not a huge deal

Reply:
I have a slight preference for the existing one, because it reads like a bullet-point instruction I would give to someone for testing this:

1. Visit the admin emoji page.
2. Delete the emoji.
3. ...